cockatrice/src/player.cpp

```diff
@@ -2657,22 +2660,27 @@ void Player::updateCardMenu(const CardItem *card)
                 }
                 cardMenu->addSeparator();
             } else if (card->getZone()->getName() == "stack") {
+                // Card is on the stack
                 cardMenu->addAction(aDrawArrow);
                 cardMenu->addSeparator();
                 cardMenu->addAction(aClone);
                 cardMenu->addMenu(moveMenu);
 
+                addRelatedCardView(card, cardMenu);
                 addRelatedCardActions(card, cardMenu);
             } else if (card->getZone()->getName() == "rfg" || card->getZone()->getName() == "grave") {
+                // Card is in the graveyard or exile
                 cardMenu->addAction(aPlay);
                 cardMenu->addAction(aPlayFacedown);
                 cardMenu->addSeparator();
                 cardMenu->addAction(aClone);
                 cardMenu->addMenu(moveMenu);
             } else {
+                // Card is in hand
```

Comment:
in hand or any custom zone that the server is using which the client isn't aware of, would be more accurate.

Reply:
True, i'll make this change